webapp/src/Service/CheckConfigService.php

```diff
@@ -337,6 +346,36 @@ public function checkTmpdirWritable()
                  $tmpdir)];
     }
 
+    public function checkHashTime()
+    {
+        $tmpuser = new User();
+        $counter = 0;
+        $time_start = microtime(true);
+        do {
+            $plainPassword = 'ryanpass';
+            $encoded = $this->passwordEncoder->encodePassword($tmpuser, $plainPassword);
+            // $user->setPassword($encoded);
+            $time_end = microtime(true);
+            $counter++;
+        } while ( ($time_end - $time_start) < 4);
+
+        $length_suggestion = 'When you have a contest duration more than a week increase the password length.';
```

Comment:
This is not related to the hashing algo right? But more a general thing.

Reply:
Yes and no, if you want more secure passwords the first thing which should be increased is the length (citing thijs) and not the hashing cost. I think i'll remove it and make something for that specific usecase?

Reply:
True. We can't really check the length of passwords, so maybe just something we should put somewhere indeed as a note.